security/fma-incident-response-improvements.md

```diff
@@ -0,0 +1,254 @@
+
+### Mitigations
+
+- Clear version documentation
+- Operator training on version differences
+- Version-specific runbooks
+- Retaining legacy `superchain-ops` tasks until further notice
+- Directing operators to reach out to OP Labs for assistance with any incident
+
+### Detection
+
+Dedicated detection apparatus not recommended. Very low likelihood of occuring in practice and OP Labs would almost certainly be made aware of any instance where these incident response capabilities are being used.
+
+### Recovery Path(s)
+
+- Immediate corrective actions (where applicable)
+- Operator retraining
+- Documentation updates
+
+### Action items
+
+- [ ] Create version-aware runbooks
+- [ ] Operator training sessions
+
+## FM3: Anchor state fails to progress within a time bound
+
+### Description
+
+The anchor state held within the `AnchorStateRegistry` could fail to progress and become too old. In particular, this could be possible if:
+
+- Guardian overuses the blacklisting/retirement mechanism to continuously block games from being used as the anchor state until the game is too old
+- Bug in `FaultDisputeGame` fails to update anchor state on game finalization
+- Bug in `AnchorStateRegistry` prevents games from updating the anchor state
+
+### Risk Assessment
+
+- Impact: HIGH
+  - Reasoning: Failure mode would block user withdrawals until resolved.
+- Likelihood: LOW/MEDIUM
+  - Reasoning: Unlikely to occur via bug or Guardian overuse in practice. Possible that this occurs if there is a bug with the `FaultDisputeGame`'s call to `setAnchorState` if no extra monitoring exists.
+
+#### Mitigations
+
+- Monitor for the age of the anchor state
+
+#### Detection
+
+- Monitor for the age of the anchor state
+
+#### Recovery Path(s)
+
+- Manual intervention to refresh the anchor state
+
+#### Action items:
+
+- [ ] Implement anchor state age monitoring
+
+## FM4: Errors in bond distribution
+
+### Description
+
+The bond distribution system could fail in multiple ways:
+
+1. Bonds not distributed (users can't withdraw)
+   - Incorrect refund recipient identification
+   - Failed distribution transactions
+   - System state preventing withdrawals
+2. Excessive bond distribution
+   - Double-counting of bonds
+   - Incorrect distribution calculations
+   - Race conditions in withdrawal process
+
+### Risk Assessment
+
+- Impact: HIGH
+  - Reasoning: Impacts bonds but does not impact tokens within the bridge.
+- Likelihood: MEDIUM
+  - Reasoning: Bond refunding logic is somewhat more complex than other changes within this upgrade.
+
+### Mitigations
+
+- Unit/integration tests
+- 3rd party audits
+- DelayedWETH `hold` functionality
+- Clear documentation of distribution rules
+- 30-minute training calls with chain operators
+
+### Detection
+
+- Existing monitoring of bonds within `op-dispute-mon`
```

Comment:
Similar comment about linking to this monitor

Reply:
https://github.com/ethereum-optimism/optimism/blob/develop/op-dispute-mon/mon/withdrawals.go

Reply:
linked in 45c4470